test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
```

Comment:
Why we need `remote_user`. Are we using that?

Reply:
No. will remove that